adot/utils/common/soak.py

```diff
@@ -0,0 +1,102 @@
+import boto3
```

Comment:
Because of this dependency, I think we should have a dockerfile to build an image that can be run to do the soak